tests/integration/test_installation.py

```diff
@@ -76,7 +76,7 @@ def test_job_failure_propagates_correct_error_message_and_logs(ws, sql_backend,
     assert len(workflow_run_logs) == 1
 
 
-@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=6))
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=12))
```

Comment:
why do we need this change? increasing a timeout would just hide problems and make tests execute slower.

Reply:
the assessment jobs runs for more than 10 mins sometimes.